chromadb/utils/embedding_functions.py

```diff
@@ -676,8 +677,17 @@ def __call__(self, input: Documents) -> Embeddings:
                 self._api_url, json={"instances": [{"content": text}]}
             ).json()
 
-            if "predictions" in response:
-                embeddings.append(response["predictions"]["embeddings"]["values"])
+            predictions = response.get('predictions')
+            if predictions:
+                for prediction in predictions:
+                    embeddings.append(prediction["embeddings"]["values"])
+            else:
+                if predictions is None:
+                    raise KeyError(f'Key `predictions` of the response from Vertex Embedding API is not existed!! Please confirm the response. \n{response}')
+                elif not isinstance(predictions, Iterable):
+                    raise TypeError(f'The response from Vertex Embedding API is not iterable!! Please confirm the response. \n{response}')
```

Comment:
We'll never hit this `elif` branch -- could we move it into the `if predictions:` branch?

Reply:
If there was a response that we expected, it means the key 'predictions' is existed. If there was an error response, it means 'predictions' is not existed.
The elif branch have to be moved into `if predictions is None`, right?

Reply:
The elif branch is based on where the predictions is None. It means that the response is error response instead of normal response.
So that's why I write down these codes.
However, I think there are some codes that could be implemented better and beetter.
Thank for your suggestion!